web/src/route.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { matchPath } from "react-router";
+import { getLocation } from "connected-react-router";
+
+export const featureDisplayPath = "/p/:projectId/f/:featureId"
+export const projectPath = "/p/:projectId"
+
+// Match selector will choose first match so put them in order.
+export const paths = [
```

Comment:
Don't think we want to export the whole list, only the base paths needed by <Route> right?

Reply:
Also, do we need to iterate over both of these paths for us to match, or would the /p/:projectId/f/:featureId also match /p/:projectId?

Reply:
We spoke - the fn returned by createMatchSelector doesn't match featureId if it's not present.